functions/lib/backup.js

```diff
@@ -18,9 +18,12 @@ function Backup (bucketName) {
       name: databaseName,
       outputUriPrefix: bucket,
       collectionIds: [
-        'whitelists',
+        'addendums',
         'agreements',
-        'addendums'
+        'appUsers',
+        'contributions',
```

Comment:
Contributions and events contain ephemeral data and they should not be backup up (unless we want to pay more for storage).

Reply:
The events collection is huge and mostly useless... I'll open a separate github issue for that.

Reply:
Done